docs/checkpoints.md

```diff
@@ -0,0 +1,26 @@
+# State checkpoints
+If enabled in the config, every _epoch length * checkpoint interval_ blocks, the node will export a state checkpoint in its datadir. This snapshot can then be imported into a brand new node to serve as a checkpoint from which to start syncing (rather than using genesis), fast-tracking the sync.
+
+## Configuration
+ * `do_checkpoints`, *boolean (default `false`)*: If `true`, enables exporting the checkpoint files. If false, checkpoints will not be created.
+ * `consensus.epochs_per_checkpoint`, *u64 (default 24)*: If `do_checkpoints` is true, determines the frequency (in epochs) at which checkpoints will be exported. This is a consensus property and will eventually be obtained on-chain rather than being configurable.
```

Comment:
> This is a consensus property and will eventually be obtained on-chain rather than being configurable.

I think we should remove this sentence. I'm not 100% confident that we want to move all of this configuration on-chain yet, so I'd rather not promise that we will.